float32 output_blindspot